recipes/turtleFSI/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "turtleFSI" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/KVSLab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 84adf1e4e7a927ee43bd8a18cab75dd08d4d5f21f4b98cac7b920bbe6525bfb9
+
+build:
+  number: 0
+  skip:
+    - True  # [win]
```

Comment:
How about instead:
```
- True  # [win or py2k]
```

Reply:
Thanks for your comment @dopplershift, I have now changed it to your suggestion.